shell/platform/tizen/channels/text_input_channel.cc

```diff
@@ -5,7 +5,6 @@
 #include "text_input_channel.h"
 
 #include <Ecore.h>
-#include <Ecore_IMF_Evas.h>
```

Comment:
We don't need this anymore?

Reply:
It seems.

cc @bbrto21

Reply:
This seems to have been included by mistake in the first place.